packages/lib/src/account/Account.test.ts

```diff
@@ -532,7 +532,7 @@ test('publishProfileOnchain', async () => {
             return profile.signedUserProfile;
         },
     );
-
+    // eslint-disable-next-line no-console
```

Comment:
I think we should either use the logger here as well or remove this line completely